### ISSUE TITLE
h2 valid style declaration showing error

### ISSUE THREAD

**Issue by salunkenagesh14** [FreeCodeCamp/FreeCodeCamp#4061]
Challenge [Waypoint: Use CSS Selectors to Style Elements](http://freecodecamp.com/challenges/waypoint-use-css-selectors-to-style-elements#?solution=%3Cstyle%3E%0A%20%20h2%7B%20color%20%3A%20blue%3B%20%7D%0A%3C%2Fstyle%3E%0A%0A%3Ch2%3ECatPhotoApp%3C%2Fh2%3E%0A%0A%3Cp%3EKitty%20ipsum%20dolor%20sit%20amet%2C%20shed%20everywhere%20shed%20everywhere%20stretching%20attack%20your%20ankles%20chase%20the%20red%20dot%2C%20hairball%20run%20catnip%20eat%20the%20grass%20sniff.%3C%2Fp%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.71 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html
<style>
  h2{ color : blue; }
</style>

<h2>CatPhotoApp</h2>

<p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>

```

**Comment by ltegman** [comment]
Confirmed that this test does not allow a space between `color` and `:`.

**Comment by ltegman** [comment]
Closed by #4079
